CHANGES.rst

```diff
@@ -1,3 +1,13 @@
+2.0.0 (2023-04-30)
+==================
+
+Fixes
+-----
+
+- Fixed a socket leak when fingerprint or hostname verifications fail. (`#2991 <https://github.com/urllib3/urllib3/issues/2991>`__)
+- Fixed error when ``HTTPResponse.read(0)`` was the first call to read the response body. (`#2998 <https://github.com/urllib3/urllib3/issues/2998>`__)
```

Comment:
Technically this is not limited to the first call, but I understand how explaining it this way makes it clearer.